dataset/dictionary/characters.json5

```
@@ -711,80 +732,96 @@
     id: "hutao",
     en: "Hu Tao",
     ja: "胡桃",
+    zhCN: "胡桃",
     pronunciationJa: "フータオ",
     tags: [ "liyue", "character-main" ],
   },
   {
     en: "Yanfei",
     ja: "煙緋",
+    zhCN: "烟绯",
     pronunciationJa: "えんひ",
+    // Pronunciataion of Chinese spelled in Japanese can be 「いぇんふぇい」, IMO. -- Bill Haku
```

OK, let's add it as a note!

```suggestion
    notes: "英語及び中国語の読みは「イェンフェイ」",
```